src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -271,6 +273,7 @@ class TWFFastDerivWrapper
 
 
 private:
+  bool is_initialized_;
```

Comment:
Please move this state and related APIs.

Reply:
Done